src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -172,31 +167,43 @@ public int getTotalMessageCount() {
   public void storeUser(User user) {
     Entity userEntity = new Entity("User", user.getEmail());
     userEntity.setProperty("email", user.getEmail());
-    
+
     Entity profileEntity = new Entity("Profile", user.getEmail(), userEntity.getKey());
     profileEntity.setProperty("email", user.getEmail());
     datastore.put(profileEntity);
   }
-  
-  /**Stores the Profile in Datastore. */
+
+  /** Stores the Profile in Datastore. */
   public void storeProfile(Profile profile) {
     Key user = KeyFactory.createKey("User", profile.getEmail());
-    
+
     Entity profileEntity = new Entity("Profile", profile.getEmail(), user);
     profileEntity.setProperty("email", profile.getEmail());
     profileEntity.setProperty("name", profile.getName());
-    /*if(profile.getProfilePicURL() != null) {
-      profileEntity.setProperty("profile_pic", profile.getProfilePicURL());
-    }*/
+    /*
```

Comment:
Might as well delete this. No reason to have commented code when you have version control

Reply:
This is just kept there so Camille works on it and it doesnt go away in master. Should I still delete it?